src/index.js

```diff
@@ -1,29 +1,46 @@
 import './style.css';
+import todoStatusUpdate from './todoStatusUpdate.js';
+import dragDrop from './dragDrop.js';
+
+const tasks = localStorage.getItem('items')
+  ? JSON.parse(localStorage.getItem('items'))
+  : [{ index: 0, description: 'Learn webpack', completed: false }, { index: 1, description: 'Read 20 pages of book', completed: false }, { index: 2, description: 'Create portfolio', completed: false }];
+
+localStorage.setItem('items', JSON.stringify(tasks));
```

Comment:
- [ ] It looks like `local storage` save the change when I check/uncheck a task, but doesn't save the new position for an element when I change the position and refresh the page (in part because the elements don't have id as I said in the other point). I kindly invite you to work on it. 🐱[Requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Store%20the%20updated%20array%20of%20items%20in%20local%20storage%2C%20so%20the%20user%20gets%20the%20correct%20list%20values%20after%20the%20page%20reloads.)